telemetry/validation/src/fix.ts

```diff
@@ -0,0 +1,28 @@
+import { reorder } from './reorder'
+import { TelemetryDefinitions, loadTelemetryDefinitions, saveTelemetryDefinitions } from './telemetryDefinitions'
+
+/**
+ * Performs automated fixes for certain validation checks on the given file.
+ * To use: call this script with the json file to fix
+ * "node fix.js foo.json"
+ */
+async function main() {
+    if (process.argv.length < 3) {
+        throw new Error('args is missing the file to validate')
+    }
+
+    const jsonPath = process.argv[2]
+    const definitions = await loadTelemetryDefinitions(jsonPath)
+
+    fix(definitions)
+
+    await saveTelemetryDefinitions(definitions, jsonPath)
+}
+
+export function fix(definitions: TelemetryDefinitions): void {
```

Comment:
Seems like we're re-inventing eslint. We should probably implement this as an eslint plugin, otherwise it's a novel system that others must learn, for not much reason.

vscode toolkit has custom eslint plugins here: https://github.com/aws/aws-toolkit-vscode/tree/master/plugins/eslint-plugin-aws-toolkits

Reply:
Apparently its a little different to make eslint rules on json. I started out that way, and opted to just write pure scripts.